src/util/request-manager.js

```diff
@@ -96,8 +96,8 @@ export default class RequestManager {
   userAgent: string;
   reporter: Reporter;
   running: number;
-  httpsProxy: ?string;
-  httpProxy: ?string;
+  httpsProxy: ?string | ?boolean;
```

Comment:
I think this type should be `string | boolean` and the default value should be `''` instead of `null` for the sake of simplicity.